src/components/StyleGuide/CardsAsNavigation.mdx

```diff
@@ -0,0 +1,199 @@
+They are used in search results and on landing pages. They can represent
+distinct items, like books and events, or signify site landing pages and
+sections. They may contain a single link or present multiple links and buttons.
+
+## Full-click functionality
+
+Navigational cards designed to link to a singular destination should be
+configured with [full-click
+functionality](/docs/components-basic-elements-card--docs#card-with-full-click-functionality),
+rendering the entire card as a clickable element.
+
+It is recommended for NYPL web apps to utilize the DS Card component, as it
+provides a hover state for the full-click functionality, thereby indicating that
+the complete card is a clickable element. To review the DS Card styling, please
+consult the <Link href="https://nypl.github.io/design-team-prototypes/full-click-card-styles/#background-image-fade" variant="external">Default background color & image fade</Link> POC.
+
+Currently, for cards not utilizing the DS Card component, identical links are
+frequently applied to both the heading and image within the card. This results
+in a scenario where two distinct elements direct to the same destination.
+Implementing full-click functionality will alleviate this redundancy.
+
+### Examples of full-click cards
+
+<Unstyled>
+  <SimpleGrid columns="4" mt="s">
+    {fullClickCard}
+    {fullClickCard}
+    {fullClickCard}
+    {fullClickCard}
+  </SimpleGrid>
+</Unstyled>
+
+<Banner
+  content={
+    <>
+      <strong>NOTE:</strong> To enable the full-click functionality in the{" "}
+      <code>
+        <strong>Card</strong>
+      </code>{" "}
+      component, pass a URL in the{" "}
+      <code>
+        <strong>mainActionLink</strong>
+      </code>{" "}
+      prop.
+    </>
+  }
+  mt="xl"
+  mb="s"
+  variant="informative"
+/>
+
+<Source
+  code={`
+  import { 
+    Card, 
+    CardContent,
+    CardHeading,
+    Image,
+  } from "@nypl/design-system-react-components";
+  
+  <Card
+    imageProps={{
+      component: (
+        <Image 
+          aspectRatio="twoByOne" 
+          border="0" 
+          src="/placeholder-photo.png" 
+        />
+      ),
+    }}
+    layout="column"
+    mainActionLink="https://nypl.org"
+  >
+    <CardHeading level="h4" size="heading7">
+      Etiam venenatis justo
+    </CardHeading>
+    <CardContent>
+      Sequi id nihil eum nemo nemo ut enim unde dolor harum rerum et accusantium.
+    </CardContent>
+  </Card>
+`}
+  language="tsx"
+/>
+
+## Distinct links
+
+Navigational cards that present links to multiple destinations or a single
+button element—excluding links that look like buttons—should not utilize
+full-click functionality.
+
+While the DS Card component permits the application of full-click functionality
+regardless of card content, NYPL patterns do not recommend this approach.
+Instead, the expectation is that each clickable element should be distinct and
+full-click functionality should not be used.
```

Comment:
Fair, but we should communicate this with Comms and others. Their expectations are that cards _should_ be full-click all the time (which is why this more-or-less came up recently).

Related to this PR, do we know if Comms and others read our storybook site? If not, how can we communicate this is our preferred approach?

Reply:
As for socializing the patterns, we will discuss this in more detail in ISW. Additionally, I would set up some time with Tatiana and Emma to review.